net/ipvsadm/patches/002-save-restore.patch

```diff
@@ -0,0 +1,36 @@
+diff -Naur a/ipvsadm-restore b/ipvsadm-restore
```

Comment:
Please consider drop this patch as it does not actually contain any functional changes.

Reply:
wow, missed the shebang line change.  sorry

Reply:
no problem